src/components/load-more-stories-base.js

```diff
@@ -74,7 +74,14 @@ export class LoadMoreStoriesBase extends React.Component {
 export class LoadMoreCollectionStories extends React.Component {
   loadMoreStories(pageNumber) {
     const stories = get(this.props, ['data', 'stories'], []);
-    return getRequest(`/api/v1/collections/${this.props.collectionSlug}`, Object.assign({}, this.props.params, {
+    const collectionType = get(this.props, ["collectionType"],null);
+    const collectionId = get(this.props, ["collectionId"],null);
+    
+    const url = (collectionType && collectionId) ? 
+                `/api/v1/${collectionType}/${collectionId}/collection`: 
```

Comment:
Is this `/api/v1/authors/<author_id>/collection` a collection api?

Reply:
Can we use `api` prop in `LoadMoreStoriesBase` for it?

Reply:
it is a collection api.